etna/articles/models.py

```diff
@@ -321,10 +323,6 @@ class FocusedArticlePage(
     The FocusedArticlePage model.
     """
 
-    author = models.CharField(
-        max_length=100, blank=True, null=True, help_text="The author of this article."
-    )
```

Comment:
@jamesbiggs Something like this should do:

```
author = models.ForeignKey("authors.AuthorPage", blank=True, null=True, related_name="focussed_articles")
```

Reply:
With the above, you should be able to do this from the author page to list the articles:
```
self.focussed_articles.live().public()
```